diagnostic.md

```diff
@@ -100,11 +130,27 @@ object.
 ```
 
 ```sh
-# your answer here
+curl --include --request PATCH http://www.example.com/donuts/1 \
+  --header "Content-Type: application/json" \
+  --data '{
+    "donuts": {
+      "olddonut": "French Cruller",
+      "newdonut": "Krüller"
+    }
+  }'
 ```
 
 Write an AJAX request to change the donut on the server using JSON.
 
 ```js
-let changeDonut = /* your answer here */;
+const changeDonut = (data) => {
+  return $.ajax({
+    url: config.apiOrigin + '/donuts/' + store.donuts.donutId,
```

Comment:
Where is `config.apiOrigin` and `store` coming from in this example? The url should be the same as the other examples